webpack/assets/javascripts/react_app/components/ConfigReports/Welcome.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import {foremanUrl, getManualURL} from '../../common/helpers'
+
+export const WelcomeConfigReports = () => {
+const description  = <> 
+{__("You don't seem to have any reports.") } <br/> 
+{__("If you wish to configure Puppet to forward it reports to Foreman, please follow")}
+<a href = {getManualURL("3.5.4PuppetReports")}> {__("setting up reporting")} </a>
+{__(" and ")}
+<a href = {"http://theforeman.org/projects/foreman/wiki/Mail_Notifications"}> {__("e-mail reporting")} </a> 
```

Comment:
I think we have a link helper to link to the wiki. In https://github.com/theforeman/foreman/pull/8184 I attempted to use it. Please consider doing to right from the start here.

If not, at least use HTTPS.
```suggestion
<a href = {"https://theforeman.org/projects/foreman/wiki/Mail_Notifications"}> {__("e-mail reporting")} </a> 
```